src/xb-opcode-private.h

```diff
@@ -89,6 +89,45 @@ xb_opcode_set_level(XbOpcode *self, guint8 level);
 guint8
 xb_opcode_get_level(XbOpcode *self);
 
+static inline gboolean
+_xb_opcode_has_flag(const XbOpcode *self, XbOpcodeFlags flag)
```

Comment:
out of interest -- why the underscore?

Reply:
I wasn't always clear to me what was public/private API since things are done with a linker script. So out of caution I just prefixed with `_` and then wrapped via the previous API in case there were external users.

Reply:
Anything that was testing the non-underscore version would continue to go through that jump through the public API entry point (which then has the inlined version inside it).

That said, if you static link tests, then you'll likely have everything inlined further via LTO.

Reply:
Gotcha, thanks.